src/js/view/view.js

```diff
@@ -858,7 +858,19 @@ define([
 
         function _onMediaTypeChange(model, val) {
             var isAudioFile = (val ==='audio');
+            var provider = _model.getVideo();
+            var isNotFlash = (provider && provider.getName().name.indexOf('flash') !== 0);
+            var previewElem = _playerElement.getElementsByClassName('jw-preview')[0];
+            var mediaElem = _playerElement.getElementsByClassName('jw-media')[0];
+
             utils.toggleClass(_playerElement, 'jw-flag-media-audio', isAudioFile);
+
+            if(isAudioFile && isNotFlash) {
```

Comment:
missing a space here, did you run jshint? If so, can you fix the .jshintrc file to prevent this type of thing?